tests/test_absoluteHeightWFS.py

```diff
@@ -0,0 +1,43 @@
+"""
+Executes an absolute height analysis for an ITL Wavefront Sensor vendor data set,
+exercising code in plot_statistics that formerly crashed on this data
+"""
+from __future__ import print_function
+import os
+import unittest
+from MetrologyData import md_factory, XyzPlane
+import MetrologyData as metData
+
+class ReferencePlaneWFSTestCase(unittest.TestCase):
+    "TestCase class for plot_statistics for Wavefront Sensor absolute height."
+    def setUp(self):
+        pass
+
+    def tearDown(self):
+        pass
+
+    def test_plot_statistics(self):
+        """
+        Verifies functionality of the generation of the histogram of absolute
+        height residuals for Wavefront Sensors.  The code previously crashed
+        if the entire range of residuals was more than 5 sigma from zero, as
+        is normally the case for Wavefront Sensors
+        """
+        infile = os.path.join(os.environ['METROLOGYDATAANALYSISDIR'], 'tests',
+                                         'WFS_vendor_metrology_data.txt')
+        dtype = 'ITL'
+        sensor_id = 'ITL-4400C-033'
+
+        sensorData = md_factory.create(infile, dtype=dtype)
+
+        # Set reference plane at znom=12.992 mm
+        #
+        sensorData.set_ref_plane(XyzPlane(0, 0, 12992.))
+
+        # Make a histogram of residual heights.
+        #
+        sensorData.plot_statistics(title='Sensor Absolute Height, %s' % infile)
+        metData.plot.save('%s_abs_height_hist.png' % sensor_id)
```

Comment:
Is it really necessary to save this plot to a file?   If the code fails, the plot would never saved, and successful unit tests shouldn't require any user inspection.   More generally, any files created by the test code should be deleted after the test has run successfully.  (This could be done in the `tearDown` method, but you would need to set the filename in `setUp` as an attribute.  In this case, I would just run `os.remove` immediately after saving the plot to a file.)  

If you absolutely need to have this plot around to inspect, it would be better to have it be written to the `tests` subdirectory explicitly.  Tools like `nosetests` and `pytest` are run from the top-level directory of the repository, and the current code will deposit this file there.